student.rb

```diff
@@ -9,4 +9,9 @@ def initialize(classroom, age, name = 'Unknown', parent_permission: true)
   def play_hockey
     '¯(ツ)/¯'
   end
+
```

Comment:
- [OPTIONAL] Check for a getter method for `@classroom`. Kindly let's add this : `attr_reader :classroom` ✍🏻 
- Note: Getter methods help you retrieve the value of an instance variable. If you don’t have a getter method, trying to retrieve a variable is going to result in an error.